src/odin/job.py

```diff
@@ -63,4 +65,20 @@ def start(self, schedule: sched.scheduler) -> None:
 
         finally:
             self.reset_tmpdir()
-            schedule.enter(run_delay_secs, 1, self.start, (schedule,))
+
+        return run_delay_secs
+
+
+def job_proc_schedule(job: OdinJob, schedule: sched.scheduler) -> None:
```

Comment:
Run all jobs in separate process. When process is finished all job resources are released.